handlers/event.js

```diff
@@ -88,6 +87,73 @@ module.exports.get = async (event, ctx, callback) => {
 
 };
 
+module.exports.getUsers = async (event, ctx, callback) => {
+
+  const id = event.queryStringParameters.id;
```

Comment:
change to eventID to avoid confusion with user id?

Reply:
All of the other `events/` endpoints use `id`. Imo we keep them the same? Do you think we should change all of them to use `eventID`?

Reply:
Yeah same, I like that